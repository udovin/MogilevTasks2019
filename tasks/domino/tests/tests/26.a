1 1 1 2 1 1 1 1 232 1 2 1 1 1 2 2 1 254 2 1 1 1 1 1 234 2 148 1 1 1 1 1 1 1 1 1 1 1 3 1 3 1 2 28 2 2 1 1 3 1 2 3 1 1 1 3 3 1 197 1 1 1 1 1 447 1 1 1 372 1 2 1 1 3 1 1 1 1 1 1 1 2 2 1 1 2 1 1 1 1 2 2 1 2 1 1 177 4 1 1 1 1 2 1 1 1 1 1 2 1 3 1 1 1 1 1 1 1 1 1 1 2 2 1 1 1 1 1 1 309 1 1 1 1 1 1 1 2 1 1 1 1 1 2 2 2 1 1 1 1 1 1 1 1 1 240 1 3 1 1 1 1 1 1 3 1 1 1 1 1 2 1 2 1 1 1 1 1 1 1 1 526 1 1 1 1 1 1 1 1 1 4 1 1 1 2 1 1 1 1 1 1 1 3 1 1 264 1 1 1 1 2 1 1 1 1 1 1 1 1 1 2 1 1 3 1 423 2 1 1 1 2 1 3 1 2 1 1 1 1 474 1 1 1 2 2 4 1 1 1 1 108 1 3 2 1 1 4 2 2 3 357 1 1 1 2 1 2 2 2 1 1 110 1 1 1 1 1 1 4 2 1 1 1 1 1 1 1 1 1 1 3 1 1 1 2 1 1 1 1 2 1 1 1 1 1 1 1 1 2 1 1 2 1 1 1 1 1 1 1 1 1 2 1 2 1 1 1 2 1 1 1 1 502 1 1 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 2 1 1 478 1 2 1 1 1 3 379 1 2 2 2 1 3 2 40 2 216 2 1 1 1 1 1 1 1 1 1 1 1 3 1 421 1 2 1 1 520 2 1 1 1 1 1 262 157 1 1 1 27 1 1 4 1 1 1 2 1 1 1 2 1 2 1 1 2 1 1 1 1 1 2 1 2 1 1 1 1 1 1 1 1 1 1 2 1 1 3 1 1 2 1 1 2 1 1 2 1 1 1 2 1 1 2 466 1 1 1 1 1 1 1 1 525 1 1 1 1 1 1 1 1 1 1 178 1 339 1 1 1 1 1 2 2 1 4 1 1 3 1 1 2 1 1 1 2 2 2 1 1 1 1 1 3 1 1 1 233 1 135 71 1 1 1 1 1 1 1 1 1 1 1 3 1 1 1 1 1 3 1 1 1 1 1 3 1 1 1 300 2 1 2 1 1 29 1 1 1 1 1 1 2 134 1 1 
