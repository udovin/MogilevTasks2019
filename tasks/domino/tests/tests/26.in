416
9176685 489011
-3223810 929965
82060046 108171
15851564 349859
-73474637 604820
-10121152 684828
26727088 515442
52283159 166730
-1436219 942433
-55325840 102386
58237076 225323
-91964694 895770
58910468 46355
21260044 242889
-20630824 917317
-13996490 626670
60799156 377111
-71523393 917679
70936477 860011
-67497080 357090
51060353 927734
-92235343 561405
18060292 169085
-26545230 443921
-57847951 102048
-78785805 860911
-7799382 664335
30743390 797240
64443645 41878
-88972568 261578
45372204 886040
-28374036 44250
15754734 465192
73142454 317770
-36611576 105828
26103734 203532
52010713 14588
-36291247 491746
31380445 243156
-24869492 239479
-82085790 84318
-93494812 40250
-33568681 760584
-76555670 200063
85437076 288332
-43298946 24788
-20462046 172152
-95740960 793671
72742795 716968
-38891719 986055
-11339469 551641
-7313073 624929
86937370 568630
-15389430 816206
-6818014 330821
48759229 910713
-71731450 21265
28099413 482910
-73258364 791647
-43561130 67551
67586789 758514
88916421 977423
-83479400 492576
42627519 469539
55706959 696164
-47384715 844065
91899402 153928
78772223 52092
79257859 969947
-16572000 572399
-42280109 78816
76736388 705563
-1172095 812878
-12085787 445617
14466854 294919
-85241796 169401
-81442992 351020
16160948 116321
-29662423 250118
7430549 209186
91497185 295756
5660713 775100
88456945 593735
-55160550 187944
38058300 280313
69417992 846603
-70241093 559507
55186801 485704
73836515 109922
-33555376 985292
1471808 615634
16829771 325834
53985271 308037
-38802058 992537
-84719255 824645
95422755 774009
1051746 77826
80620577 339298
3456199 89613
47321671 629472
-6494452 194960
-34466658 343581
-6762826 306884
18187645 888172
4855283 920528
86015114 309842
89975269 114011
-59757295 277093
-66780814 208088
-76333091 863022
-97718656 925855
-95548377 484450
66127418 798265
36918369 880373
-67728006 164322
-8475756 255227
-96295636 198746
37437813 252635
-67683989 444217
92966788 842885
-34415122 874469
-67478152 142657
65473984 718029
83973772 575401
-66203879 574656
57090272 654051
-78632783 542817
-41160545 932733
56873407 392177
-84271263 509018
-22680169 158887
-47728501 628197
-96507120 550317
45946539 907615
-71606967 248059
60253813 835622
-53455910 118239
33254854 182887
-13667229 103732
-12321609 130676
70312729 134336
-69538806 631439
-25344605 612468
20523321 384454
-36209930 316150
-90407036 156659
-99392782 524637
-46346518 693459
-99059117 516973
-23301570 521133
8534880 454378
7335188 465554
-23640798 535911
76139645 508695
-19618104 544367
49246400 847423
-80285618 768800
94002454 324823
-18080429 220178
-9694797 121795
50961598 310979
-9647304 73038
49972574 309081
46824563 50535
4204747 588685
-40360399 663842
5744585 137199
-28805016 337593
28817674 275002
24268794 565026
37963312 734460
-95312475 227231
-3114598 141113
4229847 281978
-7964081 708802
11242060 589930
-70968539 279700
-50406347 262045
42034559 136357
-4729138 235487
14119963 334896
-92236209 791334
30904872 130665
-35818985 102959
86986286 69071
-70377256 263324
-20574550 688670
79092672 281324
-24860659 351948
-57582564 846668
73873232 29069
16311201 880193
-96449923 388268
43446833 379979
57864661 966361
-82741583 175081
-78911402 679406
38207336 530243
2853170 432161
-29417282 953240
91457251 867292
40854144 92239
48954864 405882
11659134 559729
27318499 167440
93084703 857098
-16045187 478897
17976512 607181
-15075915 95638
53894405 419955
57871476 716544
-68120419 841763
69069189 735777
-65346446 22636
-4468221 756321
-36645980 899744
-47206050 725344
4522844 377322
-373840 158305
-98800678 108484
12598555 168493
-91103637 821854
-90104452 17428
-74311949 426446
89571217 669487
-82953689 673909
73327437 217450
-79098740 635445
-80183110 416694
53692655 301856
-53201554 682022
-87236900 691351
-81141375 818357
-7657413 368506
39565492 903791
-20391853 600204
-4801810 654538
97339363 209585
-33649396 476799
80510233 973455
-76156411 916151
-6574612 560908
-77113009 818678
-7521305 151345
-68947714 831211
-23549862 978212
39627543 143057
-64219494 463911
37878370 672299
-4982007 458771
12570083 544770
10705569 853202
1945863 782994
46608384 427752
-98378194 664182
14453434 124598
-17941432 986806
-39567638 905889
68802083 160929
-11697874 888848
6104847 722231
-60046059 858517
39309578 656508
29288550 232851
96491545 895781
26920503 557980
-41235637 910371
-83068563 278378
94587215 577676
-59787662 267885
-62956647 766530
10780522 356366
-33259068 6371
58324431 702794
38266123 693838
95056594 970685
9514142 519791
-6993164 802679
-48406197 528595
85173032 515906
11565266 10958
-73174971 85817
88815080 60578
-87446951 819230
-7249520 146098
-21537890 130714
-22191292 576051
59983595 153271
77133689 783276
12490125 591631
80201878 956752
-24001602 845516
57116092 681922
28824600 991385
54453998 106517
-91869016 975960
26075656 256188
-11653228 199277
53338962 332637
18246167 934151
52434508 389352
-3697304 542025
31699098 819265
-98772415 309705
-58566533 336551
52358240 541572
76095968 323846
98005553 749319
46686977 717873
85392373 35613
52591493 747344
-90318141 295431
98238451 967384
-76806730 484191
-96807782 729504
49237235 936828
16071560 202092
-36242046 23933
8033594 791314
-25533060 548665
-52488377 764593
30143576 211955
39507369 240130
-6149059 513253
84511920 910466
-46622412 743694
-67517095 710912
-41004425 826404
88902995 493298
84249268 536548
63058263 704276
23690823 366620
-10514743 361117
-95893466 879798
24395717 736487
-5193824 892698
-10186991 411098
51988539 788955
50831023 338999
-98549227 137968
91849992 603938
-27474593 214361
-81954005 302467
-95604669 546930
-97107424 521487
72764919 528199
88811255 272995
-1645073 872399
5478099 330855
98514028 878417
-55944047 919894
14941784 16108
89790115 459179
87101141 948848
-35330694 939807
-97289235 969764
-14414973 125477
-81582013 953544
-91817179 666201
86079948 898669
-75497574 316201
-74139975 814513
53452442 728920
20525554 783117
26006623 12312
26168643 470428
-74168523 309529
-11188606 505246
-24132983 138471
83041766 61500
-50167385 480348
86677267 244810
72431815 499186
-85412806 47455
-15198556 695545
62272036 256011
-43439714 829729
-72819597 452970
-79454819 350956
95886873 699679
2442734 29325
20913967 477494
49654548 572721
32254033 794909
-25769683 403130
30594792 736485
36411083 847436
-70510708 881614
1077005 882688
-64542916 783239
-929513 923639
79934644 837630
-26123825 6845
96234983 927826
12651140 789343
-52975445 651913
-84985757 233617
49946228 188089
-16948583 937593
59052921 109088
21286011 714627
28201556 65376
-34748611 134022
92411201 11535
-61593502 198527
-84447816 781452
-43081912 376331
83865927 279654
32004758 437911
80826057 709207
-48553508 382540
-20866507 290742
-40398222 113492
-88025178 978518
-90512686 468609
-98719632 632789
