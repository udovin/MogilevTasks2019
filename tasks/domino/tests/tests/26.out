2 2 1 3 4 3 3 4 4 1 3 8 1 2 4 2 1 2 1 2 1 9 3 2 1 2 11 3 1 1 6 1 4 2 1 2 1 4 1 2 1 1 3 1 1 1 2 4 4 2 2 8 3 3 5 7 1 2 3 1 1 4 4 1 1 4 1 1 6 2 1 2 3 5 1 1 2 1 3 1 1 3 8 1 3 1 1 1 2 2 3 1 1 1 3 5 5 2 1 1 1 6 4 2 5 6 1 1 1 3 7 2 1 2 5 1 1 1 4 2 5 1 2 6 1 2 1 4 3 1 1 5 3 5 3 2 1 1 1 1 1 8 3 5 1 3 14 1 13 1 1 2 4 1 1 4 2 1 2 2 2 1 1 1 8 1 1 1 3 2 4 1 1 1 12 3 1 2 1 1 3 10 1 1 1 2 3 7 1 1 1 2 2 1 5 1 3 2 1 2 5 1 6 1 1 1 1 4 1 2 4 6 2 1 1 6 3 6 1 12 2 5 1 7 3 2 1 4 1 3 3 1 1 10 2 1 3 1 4 3 2 3 6 1 7 3 1 1 5 4 3 5 2 4 8 2 1 3 1 4 1 3 4 1 2 2 5 3 7 2 1 1 1 2 1 6 1 6 1 3 1 1 1 2 1 1 1 1 1 4 4 5 1 2 1 7 3 1 5 1 2 3 3 11 1 3 2 3 3 1 1 2 2 5 4 5 2 1 2 4 1 1 3 1 4 2 3 3 5 5 1 1 5 5 1 5 4 6 3 1 3 1 1 3 5 3 7 5 4 1 3 1 2 1 8 6 1 3 1 5 1 5 4 4 1 1 6 1 6 1 1 1 5 1 2 1 3 1 1 4 1 3 3 1 5 4 3 3 4 2 2 5 1 3 1 2 1 2 3 1 1 1 1 1 1 2 1 7 2 1 2 5 2 3 4 10 
